pom.xml

```
@@ -75,6 +75,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <hdds.version>${ozone.version}</hdds.version>
     <ozone.version>0.6.0-SNAPSHOT</ozone.version>
     <ozone.release>Denali</ozone.release>
+    <declared.hdds.version>${ozone.version}</declared.hdds.version>
```

Since the property`hdds.version` is also defined, I think this one should reference it instead of `ozone.version`.

```suggestion
    <declared.hdds.version>${hdds.version}</declared.hdds.version>
```